### PR TITLE
project ready for marking

### PR DESCRIPTION
**Pull request by gingus55**
### Work Done

-Added static nav bar.
-Banner houses small bio and links to linkedin and github.
-Sidebar shows list of tools used to date.
-Main project container houses cards for projects to come.
-Site made responsive to tablet and mobile. -area of most trouble was making the aside icons list horizontally for mobile display.

### ToDo:
- more tidying of footer
- alignment in places I'm sure can be improved.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+</head>
+<body>
+<header>
+    <h1 class="name">Chris Bradshaw's Portfolio</h1>
+    <ul class="nav-links">
+        <li><a href="#about-me">About me</a></li>
+        <li><a href="#my-work">My work</a></li>
+        <li><a href="#contact">Email</a></li>
+    </ul>
+</header>
```
Comment (chelseanicholls) [inline index.html:20]: Please add spaces between sections to maximise readability

File: index.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        <div class="project-card general">
+            <img src="https://via.placeholder.com/150" alt="placeholder image" />
+            <div class="card-container">
+                <h4><b>Coming soon</b></h4> 
+                <p>.................</p>
+                <a href="">
+                    <h4>link to</h4>
+                </a>    
+            </div>
+        </div><div class="project-card general">
```
Comment (chelseanicholls) [inline index.html]: Start a new line here, you have done this a few times throughout the HTML

File: README.md [hunk 3]
```diff
@@ -1,2 +1,23 @@
...
+Site is responsive for tablets and mobiles.
+
+<h2>Image of code:</h2>
+<img src=".\assets\images\code-screenshot.png"/>
+
+<h2>Images of website:</h2>
+<img src=".\assets\images\banner-screenshot.png"/>
+<img src=".\assets\images\container-screenshot.png"/>
+
+<h3>Link to deployed page:</h3>
```
Comment (chelseanicholls) [inline README.md]: Consider moving the deployed link nearer to the top of the README

Comment (gingus55) [reply README.md]: I'll do that. Thank you :)

File: index.html (hunk 2 above)
Comment (gingus55) [reply index.html]: I will look at spacing throughout my html. Thanks

**Pull request merged**